src/services/cipher.service.ts

```diff
@@ -68,13 +70,16 @@ const DomainMatchBlacklist = new Map<string, Set<string>>([
 export class CipherService implements CipherServiceAbstraction {
     // tslint:disable-next-line
     _decryptedCipherCache: CipherView[];
+    workerThreshold: number = 250;
```

Comment:
should this be private or a const declared outside the class?

Reply:
It's only referenced in this class, so I think it can just be private.